### DISCUSSION TITLE
Student verification request is processing but also approved

### DISCUSSION THREAD

**Discussion by HridayKh** [community/community#64827] [GitHub Education] [answered]
### Hi everyone,

# My student verification request is approved but it still says it is processing at the same time
![image.png](https://github.zendesk.com/attachments/token/R1BBnVvwZDxvhY7XWzABLnFfn/?name=image.png)
![image.png](https://github.zendesk.com/attachments/token/qNL4DP8ONsmmQ3P1p0141b31e/?name=image.png)

**Comment by arafajs** [comment, marked as answer]
The processing time is within 72 hours. However, it may take longer during busy periods like the start of a new semester.
If it's been more than 72 hours and your account is still showing as "processing," you can try contacting GitHub Support for an update.

**Reply by HridayKh** [reply]
Thanks for help! :)

**Comment by Khushal942** [comment]
same issue
